target/linux/ipq806x/image/generic.mk

```diff
@@ -101,6 +113,21 @@ define Device/arris_tr4400-v2
 endef
 TARGET_DEVICES += arris_tr4400-v2
 
+define Device/aruba_ap-325
+	$(call Device/LegacyImage)
+	$(call Device/UbiFit)
+	DEVICE_VENDOR := Aruba
+	DEVICE_MODEL := AP-325
```

Comment:
You could add this as an ALT0 name for the firmware selector :)
(and maybe the Siemens OEM name too at some point)
```
DEVICE_ALT0_VENDOR := Aruba
DEVICE_ALT0_MODEL := AP-324
```